src/ProgramTest.elm

```diff
@@ -1009,12 +1009,39 @@ clickLink linkText href programTest =
         functionDescription =
             "clickLink " ++ String.Extra.escape linkText
 
-        findLinkTag =
-            [ Selector.tag "a"
-            , Selector.attribute (Html.Attributes.href href)
-            , Selector.containing [ Selector.text linkText ]
+        findLinkQuery =
+            [ ( "<a> with text"
+              , ComplexQuery.find (Just "find link")
+                    [ "link" ]
```

Comment:
If I remember correctly, this argument is a list of html tags that will be kept expanded when the ComplexQuery error renderer tries to condense down the HTML out.  So this should be `[ "a" ]`, not "link".

Reply:
Got it. I was wondering what this did.